shinyswatch/_get_theme.py

```diff
@@ -1,17 +1,12 @@
 from __future__ import annotations
 
-import os
+from ._bsw5 import BSW5_THEME_NAME
+from ._theme_utils import ShinyswatchTheme
```

Comment:
`get_theme()` and `get_theme_deps()` aren't in the same file because `get_theme()` needs `ShinyswatchTheme` and `ShinyswatchTheme` needs `get_theme_deps()`, leading to circular imports.